.github/workflows/unit-test.yaml

```
@@ -0,0 +1,27 @@
+name: Unit test
+
+on:
+    workflow_call:
+
+jobs:
+    unit-test:
+        name: Test
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)